yewtil/README.md

```diff
@@ -16,20 +16,21 @@ This should make it much easier to define simple components that don't hold stat
 * `Effect` - A way to update component state by defining what to change inside of `html!` callbacks
  instead of handling messages in `Component::update()`.
 
-This crate also has an experimental feature flag that enables the following features:
-* `Lrc` smart pointer - Rc-like pointer implemented on top of a linked list. Allows for novel state update mechanics 
+This crate also has feature flags which will enable the following **experimental** features:
+* `Lrc` smart pointer - an Rc-like pointer implemented on top of a linked list. Allows for novel state update mechanics 
 and traversal over linked shared pointers. <sup><sub>(This needs to be fuzz tested to make sure it doesn't leak.)</sub></sup>
-* DSL for `Html<Self>` - A function-based domain-specific-language for Yew that can be used in a limited capacity instead of the `html!` macro. <sup><sub>(Broken by recent changes in yew. Will be rewritten from scratch eventually.)</sub></sup>
+* This crate used to contain a domain specific language which made it possible to create components without use of the `html!` macro, but this has now been moved into a different crate (`yew-dsl`).
```

Comment:
Could you move this note to the top so that it's more obvious and also include a link?